merkletree/tb.go

```diff
@@ -1,17 +1,26 @@
 package merkletree
 
-import "github.com/coniks-sys/coniks-go/crypto/sign"
+import (
+	"bytes"
+
+	"github.com/coniks-sys/coniks-go/crypto/sign"
+)
 
 type TemporaryBinding struct {
-	Index     []byte
-	Value     []byte
-	Signature []byte
+	// IssuedEpoch would not be included in the serialization.
```

Comment:
This field would be verified by the client when it receives the TB, I thought that's enough to prevent tampering. The main reason I think this field should be included is to help the client know when it should verify the promise.